codebuild/linux-integration-tests.yml

```diff
@@ -20,6 +18,9 @@ phases:
       - echo Build started on `date`
       # Update the submodules
       - git submodule update --init
+      - export BUILDER_VERSION=$(cat .github/workflows/ci.yml | grep 'BUILDER_VERSION:' | sed 's/\s*BUILDER_VERSION:\s*\(.*\)/\1/')
```

Comment:
can you provide more info on what you are trying to solve?

Reply:
nothing really matters, just to use one single source to determine the builder version from `.github/workflows/ci.yml`. This is integration test as part of CI, so, fetch the builder version from ci.yml makes more sense.